tests/bal-tools-integration-tests/src/test/resources/projects/projectWithMultipleTools/Ballerina.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+org = "asmaj"
+name = "projectWithMultipleTools"
+version = "0.1.0"
+distribution = "2201.12.3"
+
+[[tool.openapi1]]
+id = "openapi-1"
+targetModule = "mod2"
+
+[[tool.persist1]]
+id = "persist-1"
+targetModule = "mod1"
+
+[[tool.consolidate-packages]]
+id = "consolidate-1"
```

Comment:
```suggestion
id = "consolidate-1"

```

Reply:
Addressed in [0051112](https://github.com/ballerina-platform/ballerina-lang/pull/44104/commits/0051112995b8aa8fd9bb7df0099b63fe94e3a5b1)